Thai/Cuisine/TomYumKung.md

```diff
@@ -15,4 +15,6 @@ Do you:
 
 [ดูทีวี eat Thai food and watch TV](https://www.youtube.com)
 
+[Let's go to Phuket](../travel/travel.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.